### ISSUE TITLE
[TensorRT] ERROR: INVALID_ARGUMENT: Cannot find binding of given name: input_0

### ISSUE THREAD

**Issue by niuniu1988** [NVIDIA-AI-IOT/torch2trt#227]
1、net forward code:

    def forward(self, X):
        """
        :param X:
        :return:
        """
        N = X.size()[0]

        # assert X.size() == (N, 3, self.input_size, self.input_size)

        X = self.features(X)  # extract features

        # print('X.size: ', X.size())
        # assert X.size() == (N, 512, 1, 1)

        X = X.view(N, 512, 1 ** 2)
        X = torch.bmm(X, torch.transpose(X, 1, 2)) / (1 ** 2)  # Bi-linear CNN for fine-grained classification

        # assert X.size() == (N, 512, 512)

        X = X.view(N, 512 ** 2)
        X = torch.sqrt(X + 1e-5)
        X = torch.nn.functional.normalize(X)
        X = self.fc(X)

        assert X.size() == (N, self._num_cls)
        return X

2、convert trt code:
net = Classifier().eval().cuda()
net.load_state_dict(torch.load(model_path))

# create example data
x = torch.ones(batch_size, 3, 224, 224).cuda()
# convert to TensorRT feeding sample data as input
model_trt = torch2trt(net, [x])

3、show error:
Warning: Encountered known unsupported method torch.bmm
[TensorRT] ERROR: INVALID_ARGUMENT: Cannot find binding of given name: input_0

**Comment by suwoncjh** [comment]
How did you solved the error?

**Comment by vneseresearcher** [comment]
@niuniu1988  how to solve it?
